src/server/pkg/worker/api_server.go

```diff
@@ -497,6 +506,11 @@ func HashDatum(pipelineInfo *pps.PipelineInfo, data []*Input) (string, error) {
 
 // Process processes a datum.
 func (a *APIServer) Process(ctx context.Context, req *ProcessRequest) (resp *ProcessResponse, retErr error) {
+	logger := a.getTaggedLogger(req)
+	logger.Logf("service: worker, method: Process, request: %v, response: %v, err %v, duration: %v", req, nil, nil, 0)
```

Comment:
We already have a "Received request" log message that's going to have almost all of this information in it. Also why are you logging `response` `err` and `duration` when they're always going to be `nil` here? There's basically no new information in this call compared to the `"Received request"` log below, all the information in the request is already part of the log lines.